src/jit/importer.cpp

```diff
@@ -7081,15 +7080,15 @@ GenTreePtr                Compiler::impFixupStructReturn(GenTreePtr     call,
     call->gtCall.gtReturnType = call->gtType;
 
     // Get the classification for the struct.
-    SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
-    eeGetSystemVAmd64PassStructInRegisterDescriptor(retClsHnd, &structDesc);
-    if (structDesc.passedInRegisters)
+    GenTreeCall* callPtr = call->AsCall();
+    eeGetSystemVAmd64PassStructInRegisterDescriptor(retClsHnd, &(callPtr->structDesc));
+    if (callPtr->structDesc.passedInRegisters)
     {
-        call->gtCall.SetRegisterReturningStructState(structDesc);
+        call->gtCall.SetRegisterReturningStructState(callPtr->structDesc);
```

Comment:
Is the call to SetRegisterReturningStructState() needed since we are passing it as parameter to eeGetSystemVAmd64PassStructInRegisterDescriptor() and that fills in callPtr->structDesc.  note that both callPtr->structDesc and call->gtCall.structDesc are one the same.